src/plugins/graph/adornments/adornments.tsx

```diff
@@ -10,23 +10,24 @@ import { IAdornmentModel } from "./adornment-models";
 import { useInstanceIdContext } from "../imports/hooks/use-instance-id-context";
 import { useTileModelContext } from "../imports/hooks/use-tile-model-context";
 import { useDataConfigurationContext } from "../hooks/use-data-configuration-context";
+import { DotsElt } from "../d3-types";
 
 import "./adornments.scss";
 
 export interface AdornmmentsProps {
-  dotsRef?: IDotsRef
 }
 
 export const Adornments = observer(function Adornments(props: AdornmmentsProps) {
   const
-    { dotsRef } = props,
     graphModel = useGraphModelContext(),
     dataConfig = useDataConfigurationContext(),
     instanceId = useInstanceIdContext(),
     layout = useGraphLayoutContext(),
     { isTileSelected } = useTileModelContext(),
     adornments = graphModel.adornments;
 
+  const dotsRef = useRef<DotsElt>(null);
```

Comment:
Ultimately, adornments should not need their own dots. I think the only adornment that uses dots is the connecting lines and in CODAP v3 the connecting lines have ceased to be an adornment and instead are built into the `ScatterDots` component. If this is an interim step in the process, however, that's fine.

Reply:
This passes an SVG element to ConnectingLines which is where it creates its `path`. I just left the name as `dotsRef` since that's what it was when it used the same container as the actual plot layers.

My inclination is to wait for the CODAP work on connecting-lines to be done and then make a PT issue to migrate it over to CLUE. Then we can remove this too if no other adornment needs it.